Automattic/wp-calypso
# Onboarding: move SelectItems to their own component

#### Changes proposed in this Pull Request

When migrating IntentScreen to packages the SelectItems and SelectItemsAlt components were bundled together. This PR moves them to their own component within `packages/onboarding`.

#### Testing instructions

1. Pull branch and `yarn start`
2. Go to http://calypso.localhost:3000/start/ and navigate to the intent step
3. Verify everything looks correct and no errors
4. Check http://calypso.localhost:3000/stepper as well
